koku/reporting/migrations/0328_alter_ocpusagelineitemdailysummary_monthly_cost_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.15 on 2025-02-20 13:13
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("reporting", "0327_ocpvirtualmachinesummaryp"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="ocpusagelineitemdailysummary",
+            name="monthly_cost_type",
+            field=models.TextField(
+                choices=[
+                    ("Node", "Node"),
```

Comment:
I don't really see a good reason to do the `Node_Core_Hour` as a separate database migration, so I would probably include it in this one.

Reply:
There is no migration for the hour one since its not a monthly cost type.